lightning/src/util/macro_logger.rs

```diff
@@ -32,18 +33,11 @@ macro_rules! log_pubkey {
 	}
 }
 
-pub(crate) struct DebugBytes<'a>(pub &'a [u8]);
-impl<'a> core::fmt::Display for DebugBytes<'a> {
-	fn fmt(&self, f: &mut core::fmt::Formatter) -> Result<(), core::fmt::Error> {
-		for i in self.0 {
-			write!(f, "{:02x}", i)?;
-		}
-		Ok(())
-	}
-}
+/// Logs a byte slice in hex format.
+#[macro_export]
```

Comment:
Can we actually `#[doc(hidden)]` this and all the other macros in this file that really shoulnd't be used by downstream crates but only LDK crates?

Reply:
I could see `log_internal` needing that. But which other ones? Presumably users may want to log within their trait implementations, possibly using `log_bytes`, too.